bin/cron/update_project_count

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+#
+#
+
+require 'sequel'
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require 'cdo/only_one'
+require 'cdo/properties'
+require src_dir 'database'
+require 'cdo/honeybadger'
+
+PEGASUS_DB = Sequel.connect(CDO.pegasus_db_writer.sub('mysql:', 'mysql2:'))
+STORAGE_APPS = PEGASUS_DB[:storage_apps]
+
+# This default number of lines of code is from 2019-1-21.
+DEFAULT_NUMBER_OF_PROJECTS = 35_000_000
+
+def main
+  project_count = 0
+  # Get the current amount
+  STORAGE_APPS.each do |row|
+    if row[:standalone] == true
+      project_count += 1
+    end
+  end
+
+  # check if number is bigger than previous count
+  existing_project_count = Properties.get(:metrics)['project_count'] || 0
+  if existing_project_count > project_count
+    # Log Honeybadger error that the project count somehow went down
+    command = ARGV[0]
+    stdout, stderr, status = Open3.capture3(command)
+    exitstatus = status.exitstatus
+    Honeybadger.notify_command_error "cronjob #{command} - storage_apps standalone project count decreased from #{existing_project_count} to #{project_count}", exitstatus, stdout, stderr
```

Comment:
nit: long line.

Reply:
I break up the string into its own line now but that's still kinda long. Not sure what else I should do to shorten it.

Reply:
There are a few possibilities: https://commandercoriander.net/blog/2014/11/09/a-multiline-string-cheatsheet-for-ruby/

And to make it more readable you can assign it to a variable first, and then just pass that variable into the function.